pytensor/tensor/rewriting/math.py

```diff
@@ -1567,6 +1569,18 @@ def local_op_of_op(fgraph, node):
                 return [combined(node_inps.owner.inputs[0])]
 
 
+@register_scalarize
+@node_rewriter([Sum])
+def local_sum_of_makevector(fgraph, node):
+    (array,) = node.inputs
+    if not array.owner or not isinstance(array.owner.op, MakeVector):
+        return False
+
+    values = array.owner.inputs
+    summed = aes.add(*values)
+    return [as_tensor_variable(summed)]
```

Comment:
This seems to touche on https://github.com/pymc-devs/pytensor/issues/59

Can we abstract the scalarize part from the "lift reduction operations towards the inputs", which is useful regardless of the backend?